Dockerfile

```diff
@@ -2,29 +2,25 @@
 FROM jupyterhub/k8s-hub:0.9.0-n033.h8211ad2
 
 USER root
-# install d4science auth plugin for using the service in D4Science
-RUN pip3 install git+https://github.com/enolfc/d4scienceauth.git
 
-# install oauthenticator with EGI Check-in, with refresh
-RUN pip3 install git+https://github.com/enolfc/egiauthenticator@refresh
+COPY . /egi-notebooks-hub/
 
-# install the EGI Spawner
-RUN pip3 install git+https://github.com/enolfc/egispawner
+# install the hub extensions
+RUN pip3 install /egi-notebooks-hub/
 
-RUN pip3 install git+https://github.com/enolfc/egi-hub-addons
+# install d4science auth plugin for using the service in D4Science
+RUN pip3 install git+https://github.com/enolfc/d4scienceauth.git
 
 # Customise Jupyter login
 # TODO(enolfc): this can break quite easily, should find a better way
-COPY login.html /usr/local/share/jupyterhub/templates/login.html
-COPY 401.html /usr/local/share/jupyterhub/templates/401.html
-# Again not the best, but we need a better message than "Forbidden"
-COPY 401.html /usr/local/share/jupyterhub/templates/403.html
-
-COPY egi-notebooks.png /usr/local/share/jupyterhub/static/images/jupyter.png
-COPY notebooks-logo.png /usr/local/share/jupyterhub/static/images/notebooks-logo.png
-COPY cesnet.png /usr/local/share/jupyterhub/static/images/cesnet.png
-COPY cesga.jpg /usr/local/share/jupyterhub/static/images/cesga.jpg
-COPY infn-ct.jpg /usr/local/share/jupyterhub/static/images/infn-ct.jpg
+RUN cp /egi-notebooks-hub/templates/login.html /usr/local/share/jupyterhub/templates/login.html \
+    && cp /egi-notebooks-hub/templates/401.html /usr/local/share/jupyterhub/templates/401.html \
+    && cp /egi-notebooks-hub/templates/401.html /usr/local/share/jupyterhub/templates/403.html \
```

Comment:
```suggestion
    && cp /egi-notebooks-hub/templates/403.html /usr/local/share/jupyterhub/templates/403.html \
```

Reply:
This is intended like that, we don't have a 403 template

Reply:
ops, sorry:-)